docs/creature_text_locale.md

```diff
@@ -25,15 +25,38 @@ This table is used to provide to localized clients with localized string for cre
 **Description of the fields**
 
 ### CreatureID
-Creature ID, obtained from [creature_template.entry](creature_template#entry)
+
+This is the [creature\_template.entry](creature_template#entry) to which the script is linked to.
 
 ### GroupID
-If there is more than one of the same entry (more than one text the creature says), this column is used to choose if it is a random say or an ordered list. If a creature has got more than one say text to be shown in a given order, it must be incremented for each new matching entry (ex. 0, 1, 2, 3...). If there is only one entry or only one group, this value should be 0. If there are multiple groups of texts, this value stays the same within the group while the id increments within the same group.
+
+If there is more than one of the same entry (more than one text the creature says), this column is used to choose if it is a random say or an ordered list. If a creature has got more than one say text to be shown in a given order, it must be incremented for each new matching entry (ex. 0, 1, 2, 3...). If there is only one entry or only one group, this value should be 0. If there are multiple groups of texts, this value stays the same within the group while the id increments within the same group.
+
+Example from Stormwind City Guard, creature 68:
```

Comment:
Stop with examples for simple stuff.

It just adds clutter of useless information that the descriptions already give.